mautrix_telegram/portal.py

```diff
@@ -1860,7 +1863,10 @@ async def _handle_matrix_file(
                 file = await self.main_intent.download_media(content.url)
 
             if content.msgtype == MessageType.STICKER:
-                if mime != "image/gif":
+                attributes.append(
+                    DocumentAttributeSticker(alt=content.body, stickerset=InputStickerSetEmpty())
+                )
```

Comment:
It shouldn't add the attribute for gifs (and also shouldn't set the file_name to sticker.gif for webm/tgs)

(gifs aren't stickers, it's just a hack here to allow element's gif stickers to pass through as normal gifs to telegram)